pytorch_fid/fid_score.py

```diff
@@ -39,8 +39,9 @@
 import torch
 from scipy import linalg
 from torch.nn.functional import adaptive_avg_pool2d
-
 from PIL import Image
+import torchvision.transforms as TF
+from multiprocessing import cpu_count
```

Comment:
Please move these lines to follow the correct import ordering: `torchvision` below `import torch`, `multiprocessing` below `from argparse`

Reply:
Done